src/content/en/updates/2019/04/chromium-chronicle-1.md

```diff
@@ -0,0 +1,113 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: the Chromium Chronicle, a monthly series geared specifically to Chromium developers, developers who build the browser.
```

Comment:
Can we edit this sentence to read: "the Chromium Chronicle, a monthly series geared specifically to Chromium developers—the developers who build the browser"?

Reply:
Done!